blocks/alternative_mutators/plus_minus/src/field_plus.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A field for a plus button used for mutation.
+ */
+'use strict';
+
+import * as Blockly from 'blockly/core';
+
+
+// TODO: Refactor into class?
+// TODO: Re-add plusMinus namespace prefix?
+/**
+ * Class for a plus button used for mutation.
+ * @param opt_args Arguements to pass to the 'plus' function when the button
+ *    is clicked.
+ * @constructor
+ */
+export default function FieldPlus(opt_args) {
```

Comment:
Perhaps these two should be published as separate fields. Making it more reusable in other blocks / plugins. Plus / Minus fields seems quite generic to me.

Reply:
I'm cool with that! The only caviot is that these fields do fire mutation events. So whatever is used to document them should say explicitly that they should only be used for mutation.

Reply:
Now that it's an image field with an on-click, I think you don't need a custom field type at all for it.  You just need to add the onclick handler in your block code. That matches with the fact that you're just calling the superclass constructor with the specified image, and that the onclick is mutator-specific so you wouldn't want it in a generic implementation anyway.